src/base-commands/twilio-client-command.js

```diff
@@ -5,7 +5,7 @@ const CliRequestClient = require('../services/cli-http-client');
 const { TwilioApiClient } = require('../services/twilio-api');
 const { TwilioCliError } = require('../services/error');
 const { camelCase } = require('../services/naming-conventions');
-const { HELP_ENVIRONMENT_VARIABLES, UNEXPECTED_ERROR } = require('../services/messaging/help-messages');
+const { HELP_ENVIRONMENT_VARIABLES } = require('../services/messaging/help-messages');
```

Comment:
Don't quite understand the change here. Are we not using UNEXPECTED_ERROR and NETWORK_ERROR anywhere hence don't need to import it?

Reply:
Correct.